Services/UserService/Properties/StringConstants.resx

```diff
@@ -0,0 +1,129 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Not sure for this name, it is not constants class or something like it.
It is resources file, please take a look at sample name in Kernel project